### ISSUE TITLE
After new VSCode update Remote - SSH does not read my `.profile` anymore (and R extension settings are overwritten)

### ISSUE THREAD

**Issue by jnnsbrr** [microsoft/vscode-remote-release#5852]
- VSCode Version: 1.62.1
- Local OS Version: Windows 10
- Remote OS Version: SUSE Linux Enterprise Server 12 SP3
- Remote Extension/Connection Type: SSH
- Logs: (see below)

Steps to Reproduce:

1. having the **Remote - SSH** extension installed
2. update VSCode to version 1.62.1
(3. having the [R extension](https://github.com/REditorSupport/vscode-R) installed)

and that somehow messed up the way **Remote - SSH** working as it should. It does not load my `.profile` in the terminal as it did before (I can do it manually though). And somehow also the settings.json of the R extension is overwritten (does not find R anymore). That may relates to the `.profile` issue since that loads the R module.

A colleague on another machine ran into the same bug with basically the same setup.

**Comment by tanhakabir** [comment]
Hey! This might be related to this bug https://github.com/microsoft/vscode/issues/136791 which we have a fix for out on VS Code Insiders and will be out on VS Code stable on Thursday.

Could you try connecting with VS Code Insiders to see if you still have the issue?

If you do have the issue, could you post the Remote-SSH logs from the Output channel here?

**Comment by LiveFreeAndRoam** [comment]
Installing from a local VSIX file to my remote host was successful using insiders.  I still could not install using the Extension panel, but that is possibly related to https://github.com/microsoft/vscode-remote-release/issues/5843.

Insiders version tested:
```
Version: 1.63.0-insider (user setup)
Commit: bef4dba21f68bbdb8d9f08fe04b2e1858597722b
Date: 2021-11-10T07:59:05.913Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19041
```

**Comment by tanhakabir** [comment]
I noticed in the linked issue you mentioned:

> Somehow my settings.json file on the remote machine is overwritten by the entries of the local machine (I cannot reproduce the state of how it was set before the update)

@sandy081 I'm wondering if it has anything to do with the settings sync service?

**Comment by sandy081** [comment]
I do not think so

**Comment by Piroro-hs** [comment]
WSL extension is also affected.

**Comment by tanhakabir** [comment]
As a sanity check, do you see this issue in VS Code Insiders as well?

**Comment by mika-fischer** [comment]
I'm also having the issue that `.profile` is not read anymore. This makes the extension completely unusable for me...

**Comment by mika-fischer** [comment]
It seems like not even `.bashrc` is sources. Only `.bash_env`.

Remote SSH Nightly v2021.11.13860 still has the same issue.

I've completely wiped `~/.vscode-*` on the remote host multiple times. Still no dice...

**Comment by tanhakabir** [comment]
Have you tried connecting on VS Code insiders? https://github.com/microsoft/vscode/issues/136791 might be related

**Comment by mika-fischer** [comment]
Yes most certainly https://github.com/microsoft/vscode/issues/136791 is the cause of this, thanks for the pointer. Others seem to have confirmed that the fix for 1.62.2 works, so I'll just downgrade to 1.62.0 and wait for the fixed release.

**Comment by tanhakabir** [comment]
Going to close this issue now since the fix was released earlier today on stable 1.62.2! Let me know if any of you still see this issue
